src/components/Tip.tsx

```diff
@@ -58,6 +58,7 @@ export default function Tip({ total, setTip }) {
             defaultValue={tipPercentage.toString()}
             maxLength={2}
             keyboardType="numeric"
+            testID="custom-tip-input"
```

Comment:
Is testID used for unit test? Is it possible to add this in the unit test itself instead of the component? Because if it's added here, it would be present in production, right?

Reply:
it would show, many examples online use `testID` to test `Textinput` components. I thought I would be able to just do something like `wrapper.getByRole('input')` since it's an input field but turns out that doesn't work (somehow can't find it since it's actually rendered as `TextInput` vs in a browser where it's `input`). 

From looking online, others have tried to mock out the `TextInput` component, which I tried to do so we can pass some hardcoded testID field for testing purposes but that turned out to be its own pandora's box since the approaches suggested were either:
* do jest.mock with the file path for `TextInput` in the `react-native` library which can change in the future
* mock the whole `react-native` module which turned into it's own mess and couldn't get to work

As a workaround, I did something like `wrapper.root.findByType({component name})` since the react native testing library is built on top of the `react-test-renderer` library. IMO, I don't think it would be helpful/understanding if I did a query for the component by its text value, which can change from 0 to whatever value etc.